.vscode/settings.json

```diff
@@ -0,0 +1,32 @@
+{
+    "C_Cpp.default.configurationProvider": "ms-vscode.makefile-tools",
+    "makefile.configurations": [
+        {
+            "name" : "Linux - X64 - Tiles/Sound - Release",
+            "makeArgs": [
+                "-j4",
+                "CLANG=1",
+                "CCACHE=1",
+                "NATIVE=linux64", 
+                "RELEASE=1", 
+                "TILES=1", 
+                "SOUND=1"
+            ]
+        },
+        {
+            "name" : "Linux - X64 - Tiles/Sound - Debug",
+            "makeArgs": [
+                "-j4", 
+                "CLANG=1",
+                "CCACHE=1", 
+                "NATIVE=linux64", 
+                "TILES=1", 
+                "SOUND=1"
+            ]
+        }
+    ],
+    "cmake.configureOnOpen": false,
+    "github-actions.remote-name": "upstream",
+    // NOTE: This disable the plugins formatting so astyle is used.
```

Comment:
```suggestion
    "//": "NOTE: This disable the plugins formatting so astyle is used."
```

Reply:
@ZhilkinSerg In VSCode JSON with comments are allowed in single line or multi-line format are supported.

https://code.visualstudio.com/docs/languages/json#_json-with-comments

> This mode is used for the VS Code configuration files such as settings.json, tasks.json, or launch.json. When in the JSON with Comments mode, you can use single line (//) as well as block comments (/* */) as used in JavaScript. The mode also accepts trailing commas, but they are discouraged and the editor will display a warning.